### ISSUE TITLE
API: map: getting an empty map content shouldn't return 404

### ISSUE THREAD

**Issue by Frankkkkk** [haproxytech/dataplaneapi#235]
Hi,
This issue is related to #234 .
When getting the map content of an empty map, the HTTP status code is set to 404 instead of 200.

Example of an unknown map:
```
curl -H "Content-Type: application/json" http://localhost:5555/v2/services/haproxy/runtime/maps_entries?map=NOT_EXISTS.map -v
> GET /v2/services/haproxy/runtime/maps_entries?map=NOT_EXISTS.map HTTP/1.1
> Content-Type: application/json
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Content-Type: application/json
< Vary: Origin
< Date: Tue, 25 Jan 2022 15:26:20 GMT
< Content-Length: 149
< 
{"code":404,"message":"[3]  Unknown map identifier. Please use #\u003cid\u003e or \u003cfile\u003e.\n [show map /dev/shm/NOT_EXISTS.map] not found"}
* Connection #0 to host localhost left intact
```

However, for a real, but empty, map, we get this different result:
```
curl -H "Content-Type: application/json" http://localhost:5555/v2/services/haproxy/runtime/maps_entries?map=backend.map -v
> GET /v2/services/haproxy/runtime/maps_entries?map=backend.map HTTP/1.1
> Content-Type: application/json
> 
< HTTP/1.1 404 Not Found
< Content-Type: application/json
< Content-Length: 0
```

Cheers

**Comment by mjuraga** [comment]
This has been fixed in both v2.4.6 and v2.5.1, give it a try and let us know.

**Comment by Frankkkkk** [comment]
Hi @mjuraga indeed it's perfect. 
Thanks and cheers !
